draft-bormann-cbor-yang-standin.md

```diff
@@ -140,27 +141,98 @@ Tag 100 {{-date}} can unambiguously stand in for all `date-no-zone` values.
 
 YANG type | base type | specification | stand-in
 ip-address | union | {{-yang-types}} | (see union)
-ipv4-address | string | {{-yang-types}} | tag 52
 ipv6-address | string | {{-yang-types}} | tag 54
+ipv4-address | string | {{-yang-types}} | tag 52
 ip-address-no-zone | union | RFC 6991 | (see union)
-ipv4-address-no-zone | ipv4-address | RFC 6991 | tag 52
 ipv6-address-no-zone | ipv6-address | RFC 6991 | tag 54
+ipv4-address-no-zone | ipv4-address | RFC 6991 | tag 52
 ip-address-link-local | union | {{-legacy-bis}} | (see union)
-ipv4-address-link-local | ipv4-address | {{-legacy-bis}} | tag 52
 ipv6-address-link-local | ipv6-address | {{-legacy-bis}} | tag 54
+ipv4-address-link-local | ipv4-address | {{-legacy-bis}} | tag 52
 ip-prefix | union | {{-yang-types}} | (see union)
-ipv4-prefix | string | {{-yang-types}} | tag 52
 ipv6-prefix | string | {{-yang-types}} | tag 54
+ipv4-prefix | string | {{-yang-types}} | tag 52
 ip-address-and-prefix | union | {{-legacy-bis}} | (see union)
-ipv4-address-and-prefix | string | {{-legacy-bis}} | tag 52
 ipv6-address-and-prefix | string | {{-legacy-bis}} | tag 54
+ipv4-address-and-prefix | string | {{-legacy-bis}} | tag 52
 {: title="Legacy representations in ietf-yang-types"}
 
-TO DO: Define usage of tags 54 and 52 for the cases we actually
-support.
-Addresses that have zones given cannot use tag 54/52.
-Addresses with leading zeros cannot use tag 54/52.
-Addresses that do not comply with {{RFC5952}} cannot use tag 54.
+The encoder MAY normalize IPv6 addresses and prefixes that do not comply with {{RFC5952}}
+but can be converted into the stand-in representation. The encoder implementation SHOULD be clear
+about whether this normalization is employed or not.
+
+If the schema specifies `ip-prefix`, the encoder MAY normalize prefixes with
```

Comment:
normalize with or prefixes with?  (Maybe an in-line example would clarify this.)

Reply:
clarified, hopefully